applications/nrf_desktop/configuration/nrf52840dk_nrf52840/prj_wwcb.conf

```diff
@@ -0,0 +1,211 @@
+
+CONFIG_DESKTOP_HID_REPORT_DESC="configuration/common/hid_report_desc.c"
+
+CONFIG_DESKTOP_HID_MOUSE=y
+CONFIG_DESKTOP_HID_STATE_ENABLE=y
+CONFIG_DESKTOP_HID_BOOT_INTERFACE_MOUSE=y
+
+CONFIG_DESKTOP_MOTION_BUTTONS_ENABLE=y
+CONFIG_DESKTOP_MOTION_UP_KEY_ID=0
+CONFIG_DESKTOP_MOTION_DOWN_KEY_ID=3
+CONFIG_DESKTOP_MOTION_LEFT_KEY_ID=2
+CONFIG_DESKTOP_MOTION_RIGHT_KEY_ID=1
+
+CONFIG_CAF=y
+CONFIG_CAF_PM_EVENTS=y
+CONFIG_CAF_POWER_MANAGER=y
+
+CONFIG_CAF_BUTTONS=y
+CONFIG_CAF_BUTTONS_POLARITY_INVERSED=y
+CONFIG_CAF_BUTTONS_PM_KEEP_ALIVE=n
+
+CONFIG_CAF_CLICK_DETECTOR=y
+
+CONFIG_CAF_LEDS=y
+
+CONFIG_CAF_BLE_STATE=y
+CONFIG_CAF_BLE_BOND_SUPPORTED=y
+
+CONFIG_CAF_BLE_ADV=y
+CONFIG_CAF_BLE_ADV_DIRECT_ADV=y
+CONFIG_CAF_BLE_ADV_FAST_ADV=y
+CONFIG_CAF_BLE_ADV_SWIFT_PAIR=y
+CONFIG_CAF_BLE_ADV_FAST_INT_MIN=0x0030
+CONFIG_CAF_BLE_ADV_FAST_INT_MAX=0x0060
+CONFIG_CAF_BLE_ADV_SLOW_INT_MIN=0x0070
+CONFIG_CAF_BLE_ADV_SLOW_INT_MAX=0x00a0
+
+CONFIG_DESKTOP_USB_ENABLE=y
+
+CONFIG_DESKTOP_BLE_USE_DEFAULT_ID=y
+
+CONFIG_DESKTOP_BLE_PEER_CONTROL=y
+CONFIG_DESKTOP_BLE_PEER_CONTROL_BUTTON=0x0000
+CONFIG_DESKTOP_BLE_PEER_ERASE_ON_START=y
+
+CONFIG_DESKTOP_BLE_SECURITY_FAIL_TIMEOUT_S=10
+CONFIG_DESKTOP_BLE_LOW_LATENCY_LOCK=y
+
+CONFIG_DESKTOP_CONFIG_CHANNEL_ENABLE=y
+
+CONFIG_DESKTOP_HFCLK_LOCK_ENABLE=y
+
+################################################################################
+# Zephyr Configuration
+
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=1536
+CONFIG_ISR_STACK_SIZE=1536
+CONFIG_MAIN_STACK_SIZE=768
+CONFIG_IDLE_STACK_SIZE=512
+CONFIG_BT_RX_STACK_SIZE=2048
+CONFIG_BT_HCI_TX_STACK_SIZE_WITH_PROMPT=y
+CONFIG_BT_HCI_TX_STACK_SIZE=1536
+
+CONFIG_BOOT_BANNER=n
+
+CONFIG_NUM_COOP_PRIORITIES=10
+CONFIG_NUM_PREEMPT_PRIORITIES=11
+
+CONFIG_HEAP_MEM_POOL_SIZE=512
+
+CONFIG_SYS_CLOCK_TICKS_PER_SEC=1000
+CONFIG_SYSTEM_CLOCK_NO_WAIT=y
+
+CONFIG_PM_POLICY_APP=y
+
+CONFIG_HW_STACK_PROTECTION=y
+CONFIG_RESET_ON_FATAL_ERROR=n
+
+CONFIG_GPIO=y
+
+CONFIG_SERIAL=n
+CONFIG_CONSOLE=n
+CONFIG_UART_CONSOLE=n
+
+CONFIG_HWINFO=y
+
+CONFIG_FLASH=y
+CONFIG_FLASH_PAGE_LAYOUT=y
+CONFIG_FLASH_MAP=y
+CONFIG_NVS=y
+
+CONFIG_SETTINGS=y
+CONFIG_SETTINGS_NVS=y
+
+CONFIG_REBOOT=y
+
+CONFIG_SPEED_OPTIMIZATIONS=y
+
+CONFIG_PWM=y
+
+CONFIG_LED=y
+CONFIG_LED_PWM=y
+
+CONFIG_USB_NRFX=y
+CONFIG_USB_DEVICE_STACK=y
+CONFIG_USB_DEVICE_MANUFACTURER="Nordic Semiconductor ASA"
+CONFIG_USB_DEVICE_PRODUCT="Mouse nRF52 Desktop"
+CONFIG_USB_DEVICE_VID=0x1915
+CONFIG_USB_DEVICE_PID=0x52DE
+CONFIG_USB_DEVICE_REMOTE_WAKEUP=y
+CONFIG_USB_DEVICE_HID=y
+CONFIG_USB_DEVICE_LOG_LEVEL_OFF=y
+CONFIG_USB_DRIVER_LOG_LEVEL_OFF=y
+CONFIG_USB_HID_POLL_INTERVAL_MS=1
+CONFIG_USB_HID_BOOT_PROTOCOL=y
+CONFIG_USB_HID_PROTOCOL_CODE=2
+
+CONFIG_BT=y
+CONFIG_BT_SETTINGS=y
+CONFIG_BT_SMP=y
+CONFIG_BT_SIGNING=y
+CONFIG_BT_MAX_PAIRED=2
+CONFIG_BT_ID_MAX=3
+CONFIG_BT_LL_SOFTDEVICE=y
+
+CONFIG_BT_DEVICE_NAME="Mouse nRF52 Desktop"
+CONFIG_BT_DEVICE_APPEARANCE=962
+
+CONFIG_BT_CTLR=y
+CONFIG_BT_CTLR_CONN_PARAM_REQ=n
+CONFIG_BT_CTLR_LLPM=y
```

Comment:
This one is already renamed to `CONFIG_BT_CTLR_SDC_LLPM` by 22373db0c73b4ae6704467c8f02f15ed41652082.

Reply:
Please rebase your changes to ensure your newly introduced configuration is up to date.